C/scripts/c4_ee.txt

```diff
@@ -60,3 +60,4 @@ c4client_getDoc
 c4client_start
 c4client_stop
 c4client_free
+c4client_updateDoc
```

Comment:
Should be in c4.txt, not here. None of the other c4client functions should be here either; this file is for EE-_only_ functions.

Also: I don't see the implementation of this function anywhere. You shouldn't add it to the export list before implementing it; it'll break the build with a link error.